ArduPlane/Plane.h

```diff
@@ -1044,14 +1044,16 @@ class Plane : public AP_HAL::HAL::Callbacks {
         Failsafe_Action_None      = 0,
         Failsafe_Action_RTL       = 1,
         Failsafe_Action_Land      = 2,
-        Failsafe_Action_Terminate = 3
+        Failsafe_Action_Terminate = 3,
+        Failsafe_Action_Parachute = 4
     };
 
     // list of priorities, highest priority first
     static constexpr int8_t _failsafe_priorities[] = {
                                                       Failsafe_Action_Terminate,
                                                       Failsafe_Action_Land,
                                                       Failsafe_Action_RTL,
+                                                      Failsafe_Action_Parachute,
```

Comment:
Parachute should probably be a higher priority then RTL/Land as you can't do those once you've popped a parachute. I'd guess it should be just below the flight termination level for priorities.